src/layouts/about/member.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+
+const pathToMemberImage = require.context('../../images/team');
+
+export default class Member extends Component {
```

Comment:
we can actually make this stateless function similar to submit-a-sign.jsx. but i am fine with either way. 👍